src/citrine/informatics/design_spaces/design_space.py

```diff
@@ -5,6 +5,7 @@
 from citrine._serialization.serializable import Serializable
 from citrine.informatics.modules import Module
 
+
 __all__ = ['DesignSpace']
 
 
```

Comment:
I was trying to make this work, but wasn't able to. I couldn't get the ai_resource_metadata fields to serialize, I think because of how the build methods are set up in PolymorphicSerializable and Serializable

I agree that it would be much nicer to pull the metadata into the Module level :(